detection_rules/devtools.py

```diff
@@ -733,7 +732,7 @@ def deprecate_rule(ctx: click.Context, rule_file: Path):
         ctx.exit()
 
     today = time.strftime('%Y/%m/%d')
-    deprecated_path = get_path('rules', '_deprecated', rule_file.name)
+    deprecated_path = rule_file.parent / '_deprecated' / rule_file.name
```

Comment:
Should be the correct deprecated path related to the core rules folder.

Reply:
Updated